### PR TITLE
[KHWEBA-32] delete import/export

### PR DESCRIPTION
**Pull request by gift-a**
image

### CODE REVIEW

File: src/components/toolbar/toolbar.js [hunk 1]
```diff
@@ -17,15 +17,15 @@ export class ToolbarComponent {
...
   }
 
   handleMenuClick() {
-    this.props.onMenuClicked();
+    this.props.onMenuClick();
   }
 
-  handleAboutOnclick() {
+  handleAboutClick() {
     this.props.onAboutClick();
```
Comment (ihor7016) [inline src/components/toolbar/toolbar.js]: please, could you remove this handler and move about dialog into toolbar component and handle its opening here?

Comment (gift-a) [reply src/components/toolbar/toolbar.js]: ok

**Review by Konstanty-Ivashchenko — DISMISSED** [review]

File: src/components/toolbar/toolbar.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import template from "./toolbar.html";
 
+import { AboutComponent } from "../about-dialog/about-dialog";
```
Comment (ihor7016) [inline src/components/toolbar/toolbar.js]: it should be called aboutDialogComponent

File: src/components/about-dialog/about-dialog.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.about-dialog {
+  &__header {
+    color: black;
```
Comment (ihor7016) [inline src/components/about-dialog/about-dialog.scss:3]: should it be black? please look at styles of other dialogs and make it consistent -- mb we do not need this style

Comment (gift-a) [reply src/components/about-dialog/about-dialog.scss:3]: It is black in other dialogs. An toolbar rewrites this color to white

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**